cli/src/main/java/org/verapdf/cli/commands/CommandVeraPDF.java

```diff
@@ -0,0 +1,53 @@
+package org.verapdf.cli.commands;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+/**
+ * This class holds all command line options used by VeraPDF application.
+ * @author Timur Kamalov
+ */
+@Parameters(commandNames = "verapdf")
+public class CommandVeraPDF extends Command {
+
+    @Parameter(names = "-validate", required = true)
```

Comment:
I vote that we should use the [GNU conventions](https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html) where "word" parameters are prefixed by a double hyphen, i.e. ```--input``` rather than ```-input```. SIngle hyphens can be used for single character abbreviations of word options, e.g. ```-i``` as an alternative to ```--input```.